src/test/java/api/support/ApiTests.java

```diff
@@ -13,7 +13,7 @@
 
 public abstract class ApiTests {
   private static boolean runningOnOwn;
-  protected final OkapiHttpClient okapiClient;
+  protected static OkapiHttpClient okapiClient;
```

Comment:
Why does this need to change to be static?

Reply:
I need a to use a parameterized test, but it doesn't work when initializing `okapiClient` in constructor (NPE on class init step). Have to initialize it in @BeforeClass method, which operates with static context only.